tests/baselines/reference/promisePermutations3.errors.txt

```diff
@@ -79,7 +79,7 @@ tests/cases/compiler/promisePermutations3.ts(159,21): error TS2345: Argument of
           Type '(value: number) => Promise<any>' is not assignable to type '(value: string) => any'.
             Types of parameters 'value' and 'value' are incompatible.
               Type 'number' is not assignable to type 'string'.
-tests/cases/compiler/promisePermutations3.ts(165,21): error TS2345: Argument of type '{ <T>(x: T): IPromise<T>; <T>(x: T, y: T): Promise<T>; }' is not assignable to parameter of type '(value: (x: any) => any) => Promise<any>'.
+tests/cases/compiler/promisePermutations3.ts(165,21): error TS2345: Argument of type '{ <T>(x: T): IPromise<T>; <T>(x: T, y: T): Promise<T>; }' is not assignable to parameter of type '(value: {}) => Promise<any>'.

```

Comment:
This change seems.... wrong. I don't see anything which would intentionally prompt the change.